target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -69,6 +69,7 @@ ramips_setup_interfaces()
 	dir-320-b1|\
 	dir-610-a1|\
 	dir-615-h1|\
+	dwr-512-b|\
```

Comment:
Please add the following to your dts file:

&esw {
  mediatek,portmap = <0x2f>;
};

0x2f is a binary mask (00010 1111), which sets the lan/wan port property and the vlans of the switch. The post right number is port0 and 1 means lan port. Check the output of ``swconfig dev switch0 show`` to see what I'm talking about.

It should be save to drop this line afterwards. Have a look how it is done for the dir-620-a1.